src/Elements/Geometry/Profile.cs

```diff
@@ -70,6 +70,17 @@ public void Transform(Transform t)
                 this.Voids[i].Transform(t);
             }
         }
+        /// <summary>
+        /// Return a new profile that is this profile scaled about the origin by the desired amount.
+        /// </summary>
+        public Elements.Geometry.Profile Scale(double amount)
+        {
+            var transform = new Elements.Geometry.Transform();
+            transform.Scale(amount);
+            var transformedProfile = transform.OfProfile(this);
+
+            return new Elements.Geometry.Profile(transformedProfile.Perimeter, transformedProfile.Voids, Guid.NewGuid(), transformedProfile.Name);
```

Comment:
Is this necessary? Can you just do `return transform.OfProfile(this)`?

Reply:
good point.  the only thin missing now is that `transform.OfProfile()` doesn't maintain the name of the profile, but maybe we don't think of profiles as elements that need names?